style.css

```diff
@@ -0,0 +1,12 @@
+body {
+    background-color: aqua;
+}
+#title{
+    text-align: center;
+}
+#description {
+    text-align: center;
+}
+#survey-form{
+    text-align: center;
+}
```

Comment:
I suggest you to put a space after each block and you need one  space after name of the class or id 
e.g 
#survey-form { ......}